pkgs/by-name/li/linien-client/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, python3
+, fetchFromGitHub
+, linien-common
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "linien-client";
+  version = "0.8.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
```

Comment:
I haven't done anything like that with by-name, but is it possible to extract common ( repetitive bits ) into something like pkgs/by-name/li/linien-common/common.nix ?

Reply:
maybe?
```
 .
└──  pkgs
    └──  by-name
        └──  li
            └──  linien
                ├──  client.nix
                ├──  common.nix
                ├──  gui.nix
                ├──  server.nix
                └──  test.nix
```

Reply:
i am not familiar with by-name loading code, will it only load `package.nix` ?

Reply:
I can also transfer it to other than with pkgs/by-name. Not sure where to put it though